src/Servers/IIS/IIS/test/Common.FunctionalTests/Http2Tests.cs

```diff
@@ -643,9 +617,6 @@ public async Task Reset_AfterResponseHeaders_Resets()
                         Assert.Equal("200", decodedHeaders[HeaderNames.Status]);
                     });
 
-                    var dataFrame = await h2Connection.ReceiveFrameAsync();
-                    Http2Utilities.VerifyDataFrame(dataFrame, expectedStreamId: 1, endOfStream: false, length: 0);
```

Comment:
Why didn't this have the `if (Environment.OSVersion.Version >= Win10_Regressed_DataFrame)` check like the other tests?

Reply:
Probably because it was added later and only tested on a version > Win10_Regressed_DataFrame. We don't have automated test coverage for builds within the failing range, only manual coverage.